applications/nrf_desktop/src/util/CMakeLists.txt

```diff
@@ -5,3 +5,19 @@
 #
 target_sources_ifdef(CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE app
 			PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/config_channel.c)
+
+if(CONFIG_DESKTOP_BLE_QOS_ENABLE)
+	if(CONFIG_FLOAT)
```

Comment:
indentation is two spaces.

Reply:
Please also use IS_ENABLED instead of preprocessor whenever possible

Reply:
? Note that this is CMake code.